docs/data-modeling-principles.md

```diff
@@ -0,0 +1,115 @@
+name concepts and how the concepts should be related to each other.
+Having real use cases is what grounds the modeling effort in reality and ensures that the data model is useful
+for the organization.
+
+### The essence of data modeling is **cooperation**
+
+Data modeling is how people with different backgrounds and perspective come together to create a shared understanding
+of the business and data. You should expect that there will be friction in this process as it surfaces and makes
+concrete the understanding from different parts of the organization. It is by working through this friction that
+the organization creates the shared understanding that unlocks better collaboration across the organization.
+
+### Parsimonious Model
+
+The principle of parsimony is a fundamental principle in science and engineering. It states that a model should be
+as simple as possible, but not simpler. In the case of an Enterprise Data Model, it should contain all shared
+concepts across the organization, but no concepts that are only relevant for a single business area.
+
+### Create one main data model, the Enterprise Data Model
+
+This follows from the principle of cooperation. By having one main data model, it forces the organization to
+cooperate. Once the Enterprise Data Model is in place, it serves as the foundation for cooperation across the
+organization. It also explicitly encodes implicit knowledge, making it clearer and easier to discuss,
+onboard new employees and partners, and to make decisions.
+
+### Each business area can create one or more distributed data models
+
+This follows from the principle of cooperation and parsimonious model. The development of the
+Enterprise Data Model will necessarily be a slower process, and it typically will become a larger data model
+that can be challenging to use for practical applications. A Solution Data Model is a distributed data model
+using a subset of the Enterprise Data Model, which can also be extended with additional concepts that are
+not part of the Enterprise Data Model. This allows business areas to move faster and to adapt the data model
+to their specific needs. It is also a typical way to test new concepts before they are integrated into
+the Enterprise Data Model.
+
+### Solution Data Models should always be referencing the Enterprise Data Model
+
+This follows from the principle of cooperation and parsimonious model. The trade-off that the Enterprise and
+Solution data models are trying to balance is between *cooperation* and the *ability to move fast*. If the organization
+only had a single data model, development would be extremely slow and many concepts would be irrelevant for
+most parts of the organization. On the other hand, if each business area had its own data model,
+it would be very challenging to cooperate across the organization, a situation which we refer to as *siloing*.
```

Comment:
```suggestion
it would be very challenging to cooperate across the organization, a situation which we refer to as *siloing* as data, information and knowledge among various business units would be isolated.
```